content/administration/on_premise/deploy.rst

```diff
@@ -600,6 +600,141 @@ It should be stored securely, and should be generated randomly e.g.
 
 which will generate a 32 characters pseudorandom printable string.
 
+Reset the master password
+-------------------------
+
+There may be instances where the master password is misplaced or compromised and needs to be reset.
+The following process is for system administrators of an Odoo on-premise database to manually reset
+and re-encrypt the master password.
+
+.. seealso::
+   For more information about changing an Odoo.com account password, see this documentation:
+   :ref:`odoocom/change_password`.
+
+When creating a new on-premise database, a random master password is generated. Odoo recommends
+using this password to secure the database. This password is implemented by default so there is a
+secure master password for any Odoo on-premise deployment.
+
+.. warning::
+   When creating an Odoo on-premise database the installation is accessible to anyone on the
+   internet until this password is set to secure the database.
+
+The master password is specified in the Odoo configuration file (`odoo.conf` or `odoorc` (hidden
+file)). The Odoo master password is needed to modify, create, or delete a database through the
+graphical user interface (GUI).
+
+Locate configuration file
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First, :guilabel:`Open` the Odoo configuration file (`odoo.conf` or `odoorc` (hidden file)).
```

Comment:
unless there is a button/link titled, "Open" at this point (and present on the interface) I don't think a GUI label is necessary in this instance.